Hoxtygen/next-fullstack
# feat: implement access refresh token(HOM58)

## What does this PR do?
Implements refresh token

## How should this be manually tested?
* Clone the app
* Open a terminal on your PC and run `git clone <repo_url>`
* `cd` into the project directory and run `yarn install`.
* After package installation run `yarn dev`
* Open a tab on your browser and navigate to `http://localhost:3000/auth/signup` to sign up  or `http://localhost:3000/auth/login` to log in if you already have an account
* On the dashboard, create as many listings as possible
* Access token expires after 30 minutes. Due to implementation of refresh token you'll not notice access token expiration.



## What are the relevant Jira board stories?
[HOM-58](https://wasiu-dev.atlassian.net/jira/software/projects/HOM/boards/2?selectedIssue=HOM-58)

[HOM-58]: https://wasiu-dev.atlassian.net/browse/HOM-58?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
